packages/frameworks-sveltekit/package.json

```diff
@@ -51,7 +51,7 @@
   },
   "peerDependencies": {
     "@sveltejs/kit": "^1.0.0 || ^2.0.0",
-    "svelte": "^3.54.0 || ^4.0.0" || "^5"
+    "svelte": "^3.54.0 || ^4.0.0 || ^5"
```

Comment:
@balazsorban44 

Currently trying out the new Svelte 5 alongside with @auth/sveltekit.
I am currently using "svelte": "^5.0.0-next.69" but it's complaining about peerDependency still.
```
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"^3.54.0 || ^4.0.0 || ^5" from @auth/sveltekit@0.13.0
```

^5.0.0-next.69 should be working if ^5 is specfied right?